nbgrader/docs/source/user_guide/advanced.rst

```diff
@@ -11,71 +11,6 @@ Running nbgrader with JupyterHub
 
 Please see :doc:`/configuration/jupyterhub_config`.
 
-.. _assignment-list-installation:
```

Comment:
Ahh, yes the way I wrote it wasn't clear. Yes, both installation and activation can be done for all users. I will clarify that.